doc/ale-gitcommit.txt

```diff
@@ -0,0 +1,41 @@
+===============================================================================
+ALE Git Commit Integration                              *ale-gitcommit-options*
+
+
+===============================================================================
+gitlint                                                 *ale-gitcommit-gitlint*
+
+g:ale_gitcommit_gitlint_executable         *g:ale_gitcommit_gitlint_executable*
+                                           *b:ale_gitcommit_gitlint_executable*
+  Type: |String|
+  Default: `'gitlint'`
+
+  This variable can be changed to modify the executable used for gitlint.
+
+
+g:ale_gitcommit_gitlint_options               *g:ale_gitcommit_gitlint_options*
+                                              *b:ale_gitcommit_gitlint_options*
+  Type: |String|
+  Default: `''`
+
+  This variable can be changed to add command-line arguments to the gitlint
+  invocation.
+
+  For example, to dinamically set the gitlint configuration file path, you
+  may want to set >
+
+  let g:ale_gitcommit_gitlint_options = '-C /home/user/.config/gitlint.ini'
+<
+g:ale_gitcommit_gitlint_use_global         *g:ale_gitcommit_gitlint_use_global*
```

Comment:
Put two blank lines after option sections. The line with `<` can be considered blank, as the character is concealed.

Reply:
Ok. Done.